packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -88,14 +104,19 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
   );
 };
 
-type Props = ConnectedProps & { isInitiallyDisabled: boolean };
-
-const RichTextEditor = (props: Props) => {
-  const { sdk, isInitiallyDisabled, onAction, restrictedMarks, ...otherProps } = props;
+const RichTextEditor = (props: RichTextProps) => {
+  const { sdk, isInitiallyDisabled, onAction, restrictedMarks, onChange, ...otherProps } = props;
   const isEmptyValue = React.useCallback(
     (value) => !value || deepEquals(value, Contentful.EMPTY_DOCUMENT),
     []
   );
+  // TODO: Deprecated use of onChange, should be removed in next major version
+  // To prevent breaking change, assigning onChange to SDK handler
+  React.useEffect(() => {
+    if (onChange) {
+      sdk.field.onValueChanged(onChange);
```

Comment:
we should unsubscribe here on unmount

Reply:
`onValueChanged` returns an unsubscribe function already

Reply:
True 😅  forgot about that